addons/dragging/functions/fnc_setCarryable.sqf

```diff
@@ -56,8 +56,25 @@ GVAR(initializedClasses_carry) = _initializedClasses;
 
 private _icon = [QPATHTOF(UI\icons\box_carry.paa), QPATHTOF(UI\icons\person_carry.paa)] select (_object isKindOf "CAManBase");
 
-private _carryAction = [QGVAR(carry), LLSTRING(Carry), _icon, {[_player, _target] call FUNC(startCarry)}, {[_player, _target] call FUNC(canCarry)}] call EFUNC(interact_menu,createAction);
-private _dropAction = [QGVAR(drop_carry), LLSTRING(Drop), "", {[_player, _target] call FUNC(dropObject_carry)}, {[_player, _target] call FUNC(canDrop_carry)}] call EFUNC(interact_menu,createAction);
+private _carryAction = [
```

Comment:
Since nothing has changed here, we can revert these changes, unless the new formatting is preferred.

Reply:
Meh. It's more readable this way IMO